src/SingleProject/Resizetizer/src/GenerateSplashAndroidResources.cs

```diff
@@ -65,7 +65,14 @@ void WriteDrawable(ResizeImageInfo splash)
 			writer.WriteAttributeString("xmlns", "android", ns: null, value: Namespace);
 
 			writer.WriteStartElement("item");
-			writer.WriteAttributeString("android", "drawable", Namespace, "@drawable/" + splash.OutputName);
+			writer.WriteAttributeString("android", "width", Namespace, "108dp");
+			writer.WriteAttributeString("android", "height", Namespace, "108dp");
```

Comment:
Do we know the size of the original image here? Could we do some math to get a width/height that wouldn't stretch the image? Basically make the larger dimension 108dp, and use the ratio for the other dimension.

Reply:
I have this logged in #5676

It is going to require a bit more work, and we might as well fix everything correctly using the proper versioned drawables.

Reply:
* Tested on Android 21 and the splash is still dodgy as it always was: #5676 
* I will re-evaluate that in the same issue as iOS and does different things, and Android before 12 does actually have more space. I will add a comment to the issue.

Reply:
Fixed in https://github.com/dotnet/maui/pull/10516